packages/expo-location/android/src/main/java/expo/modules/location/LocationModule.java

```diff
@@ -3,7 +3,10 @@
 package expo.modules.location;
 
 import android.Manifest;
+import android.app.Activity;
```

Comment:
Do you see any opportunity to extract some parts of this file to separate classes? 913 lines is kind of a lot… 🤷‍♂️